src/main/java/uk/gov/hmcts/reform/professionalapi/service/impl/OrganisationServiceImpl.java

```diff
@@ -233,6 +215,49 @@ public OrganisationsDetailResponse retrieveOrganisations() {
         return updatedOrganisationDetails;
     }
 
+    @Override
+    public OrganisationsDetailResponse retrieveAllOrganisations() {
+        List<Organisation> retrievedOrganisations = organisationRepository.findAll();
+
+        if (retrievedOrganisations.isEmpty()) {
+
+            log.info("No Organisations Retrieved...");
```

Comment:
this logger has no value as anyways the exception is thrown afterwards which should print the stack trace in logs